xml/planning-planning-hw_support_software.xml

```diff
@@ -22,23 +22,8 @@
         </itemizedlist>
         <para>The following are the requirements for your vCenter server:</para>
         <itemizedlist>
-                <listitem>
-                        <para>Software </para>
-                        <itemizedlist>
-                                <listitem>
-                                        <para>vCenter 5.5 Update 3 and above (It is recommended to
-                                                run the same server version as the ESXi
-                                                hosts)</para>
-                                </listitem>
-                        </itemizedlist>
-                </listitem>
-                <listitem>
-                        <para>License Requirements </para>
-                        <itemizedlist>
-                                <listitem>
-                                        <para>vSphere Enterprise Plus license</para>
-                                </listitem>
-                        </itemizedlist>
-                </listitem>
+                <listitem><para>Software: vCenter 5.5 Update 3 and above (It is recommended to run the
```

Comment:
Nice to have: you could use formalpara: `<formalpara><title>Software><para>vCenter 5.5 ...</para></formalpara>